tests/queries/0_stateless/03203_system_query_metric_log.reference

```diff
@@ -0,0 +1,6 @@
+1	1
```

Comment:
This reference results are extremely obscure, so when it fails (because it will fail at some point) it will be impossible to understand why.

Reply:
This should make it easier to understand: https://github.com/pamarcos/ClickHouse/blob/system-query-metric-log/tests/queries/0_stateless/03203_system_query_metric_log.reference

```
number_of_metrics_1000_ok	timestamp_diff_in_metrics_1000_ok
number_of_metrics_1234_ok	timestamp_diff_in_metrics_1234_ok
number_of_metrics_123_ok	timestamp_diff_in_metrics_123_ok
0
0
3
```